package/gluon-web-admin/i18n/de.po

```diff
@@ -17,7 +17,7 @@ msgid "Advanced settings"
 msgstr "Erweiterte Einstellungen"
 
 msgid ""
-"Alternatively, you can set a password to access you node. Please choose a "
+"Alternatively, you can set a password to access your node. Please choose a "
```

Comment:
Sure, this is unrelated, but it only affects files, that we change in this PR anyway. Should I still split it off?